app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,47 @@
+class MoviesController < ApplicationController
+
+  def index
+    movies = Movie.all
+    render json: jsonify_index(movies), status: :ok
+  end
+
+  def show
+    movie_id = params[:id]
+    movie = Movie.find_by(id: movie_id)
+
+    if movie
+      render json: jsonify_show(movie), status: :ok
+    else
+      render_error(:not_found, {id: ["Movie with id: #{movie_id} not found"]})
+    end
+  end
+
+  def create
+    movie = Movie.new(movie_params)
+
+    if movie.save
+      render json: {id: movie.id}
+    else
+      render_error(:bad_request, movie.errors.messages)
+    end
+  end
+
+  private
+
+  def movie_params
+    # 11 tests are failing and found out that we need to change the params from
+    # params.require(:movie)
+    # now 8 tests passed on postman
```

Comment:
What is this comment in reference to so I can respond to it?

Reply:
Oh these were our note for ourselves but we forgot to delete it in the final version. All of our tests pass. Sorry for the confusion.